### ISSUE TITLE
TDI.Ajax.Request.send should support sending parameters in terms of FormData

### ISSUE THREAD

**Issue by msevcenko** [twinstone/tdi#35]
Sometimes it is required to send data in POST reques using TDI.Ajax.Request.send, which may be potentially large, and would exceed server limit (POST requests are sent url-encoded by default).

multipart/form-request encoding should be also supported. If FormData object is passed to the send method, the call fails. Send method internally uses jquery ajax call, which requires to pass processData = false and contentType = false parameters to successfully process FormData parameters.

**Comment by msevcenko** [comment]
Suggested fix:

```
if (HAS_FORMDATA_SUPPORT && options.data instanceof FormData) {
	options.processData = false;
	options.contentType = false;
}
```

**Comment by centi** [comment]
Fixed in [v1.10.0](https://github.com/twinstone/tdi/releases/tag/v1.10.0)
